packages/tc-ui/src/primitives/relationship/lib/view-relationship.jsx

```diff
@@ -1,15 +1,16 @@
 const React = require('react');
+const { sortBy } = require('lodash');
```

Comment:
this will be used on the client side. To make the code bundel smaller we can instead `npm install lodash.sortby` then `sortBy = require('lodash.sortby');`